.eslintrc.json

```diff
@@ -1,19 +1,13 @@
 {
-  "plugins": ["node", "prettier"],
+  "plugins": ["prettier"],
   "parserOptions": {
     "sourceType": "module"
   },
   "env": {
```

Comment:
This `env` is added by `eslint-config-standard`, i.e. this could be removed.

https://github.com/standard/eslint-config-standard/blob/3f4a3fed2c5d5fdf03e5c32e6c87d2fdc3ea4282/eslintrc.json#L10-L13

Same thing with `"sourceType": "module"`